test/test_graph.py

```diff
@@ -53,6 +58,12 @@ def test_from_adjlst_graph(self):
         self.g = DenseGraph.from_adjlst_graph(AdjlstGraph.from_mat(MAT, IDS))
         self.validate()
 
+    def test_properties(self):
+        self.g = DenseGraph.from_mat(MAT, IDS)
+        self.assertEqual(self.g.num_nodes, 3)
```

Comment:
Make sure either 1) ``AdjlstGraph`` produce the correct results (need to implement ``num_edges``, or 2) throws error when called with ``AdjlstGraph``.